test/k8sT/Updates.go

```diff
@@ -253,9 +253,9 @@ func InstallAndValidateCiliumUpgrades(kubectl *helpers.Kubectl, oldHelmChartVers
 		}
 		if helpers.RunsWithoutKubeProxy() || helpers.RunsOn419Kernel() {
 			opts["global.nodePort.device"] = privateIface
-			if helpers.RunsOn419Kernel() {
-				opts["global.debug.enabled"] = "false"
-			}
+		}
+		if helpers.RunsOnNetNextOr419Kernel() {
+			opts["global.debug.enabled"] = "false"
```

Comment:
Isn't the same change required for the third `opts["global.debug.enabled"] = "false"` case below (around line 419)?

Reply:
That case around line 419 is just the preflight check, I don't think this flag has any effect there.

Reply:
Although on net-next we have `helpers.RunsWithoutKubeProxy()` on true, which may be why this didn't have an effect compared to prior runs.